run/local-cluster.sh

```diff
@@ -75,7 +75,7 @@ DNS_FILE="$DNS_DIR/addn-hosts"
 
 # Start dnsmasq container. We wait in a loop until the DNS additional
 # hosts file appears before starting the dnsmasq process.
-DNS_CID=$(docker run -d -v "$DNS_DIR:/dnsmasq.hosts" --name=$DNSMASQ_NAME $DNSMASQ_IMAGE /bin/sh -c "while true; do if [ -f /dnsmasq.hosts/addn-hosts ]; then break; else echo 'waiting 1s for DNS address info...'; sleep 1; fi; done; /usr/sbin/dnsmasq -d")
+DNS_CID=$(docker run -d -v "$DNS_DIR:/dnsmasq.hosts" --name=$DNSMASQ_NAME $DNSMASQ_IMAGE /bin/sh -c "while true; do if [ -f /dnsmasq.hosts/addn-hosts ]; then break; else echo 'waiting 1s for DNS address info...'; sleep 1; fi; done; cat /dnsmasq.hosts/addn-hosts; cat /etc/resolv.dnsmasq.conf; /usr/sbin/dnsmasq -d")
```

Comment:
You never get to see this unless something goes wrong, in which case you want to see that - so I'd leave it in.